pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -999,6 +1058,12 @@ def force_gnd(self):
         ``RZ`` (not implemented). (``DZ``)"""
         self.write("DZ %d" % self.channel)
 
+    def restore_settings(self):
+        """Restores the settings of the channel to the state before
+        using `force_gnd`. (``RZ``)
+        """
+        self.write("RZ %d" % self.channel)
```

Comment:
Why don't you use a more convenient f-string:

```suggestion
        self.write(f"RZ {self.channel}")
```